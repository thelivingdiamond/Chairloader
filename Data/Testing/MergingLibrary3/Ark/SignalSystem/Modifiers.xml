<?xml version="1.0"?>
<modifiers>
    <modifier
        name="Modifier1"
        id="11111">
        <rule
            ch:index="1000">
            <inputFilters />
            <outputEffects>
                <effect
                    signalID="9999"
                    scale="123" />
            </outputEffects>
        </rule>
    </modifier>
    <modifier
        name="Modifier2"
        id="22222">
        <rule
            ch:index="1000">
            <inputFilters>
                <matchingKey
                    signalID="8888" />
            </inputFilters>
            <outputEffects>
                <effect
                    signalID="7777"
                    scale="0.5" />
            </outputEffects>
        </rule>
        <rule
            ch:index="2000">
            <inputFilters>
                <matchingKey
                    signalID="7878" />
            </inputFilters>
            <outputEffects>
                <effect
                    signalID="9898"
                    scale="25.25" />
            </outputEffects>
        </rule>
    </modifier>
</modifiers>
