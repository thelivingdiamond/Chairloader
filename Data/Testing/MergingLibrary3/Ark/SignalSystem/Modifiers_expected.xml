<?xml version="1.0"?>
<modifiers>
    <modifier
        name="Modifier1"
        id="11111">
        <rule
            ch:index="1000">
            <inputFilters />
            <outputEffects>
                <effect
                    signalID="9999"
                    scale="123" />
            </outputEffects>
        </rule>
    </modifier>
    <modifier
        name="Modifier2"
        id="22222">
        <rule
            ch:index="1000">
            <inputFilters>
                <matchingKey
                    signalID="8888" />
            </inputFilters>
            <outputEffects>
                <effect
                    signalID="7777"
                    scale="0.5" />
            </outputEffects>
        </rule>
        <rule
            ch:index="2000">
            <inputFilters>
                <matchingKey
                    signalID="7878" />
            </inputFilters>
            <outputEffects>
                <effect
                    signalID="9898"
                    scale="99999" />
            </outputEffects>
        </rule>
        <rule
            ch:index="2050">
            <inputFilters>
                <matchingKey
                    signalID="45456" />
            </inputFilters>
            <outputEffects>
                <effect
                    signalID="64654"
                    scale="789" />
            </outputEffects>
        </rule>
    </modifier>
    <modifier
        name="Modifier3Mod"
        id="33333">
        <rule
            ch:index="1000">
            <inputFilters />
            <outputEffects>
                <effect
                    signalID="9999"
                    scale="123" />
            </outputEffects>
        </rule>
    </modifier>
</modifiers>
